bench/BenchCSV.hs

```diff
@@ -365,6 +359,8 @@ benchDListUtf8 = bench "utf8 + renderTableD maxiTable" $
 -- Text Builder
 ------------------------------------------------------------------------------
 
+#ifdef MIN_VERSION_text
```

Comment:
Could you add a comment that explains why these benchmarks are hidden behind CPP?